src/OpenConext/EngineBlockApiClientBundle/Value/ConsentListFactory.php

```diff
@@ -109,8 +109,7 @@ private static function createServiceProvider($data)
         Assert::keyExists($data, 'eula_url', 'Consent JSON structure must contain key "eula_url"');
         Assert::keyExists($data, 'support_email', 'Consent JSON structure must contain key "support_email"');
         Assert::keyExists($data, 'name_id_format', 'Consent JSON structure must contain key "name_id_format"');
-        Assert::keyExists($data, 'support_url_en', 'Consent JSON structure must contain key "support_url_en"');
-        Assert::keyExists($data, 'support_url_nl', 'Consent JSON structure must contain key "support_url_nl"');
+        Assert::keyExists($data, 'support_url', 'Consent JSON structure must contain key "support_url"');
```

Comment:
Could you not be even more specific here? Yes, the support_url key must exist but it should also be an array, with one of keys: `en` or `nl`.

Reply:
Above mentioned situation could easily be tested in the ConsentListFactoryTest

Reply:
In practice, yes, but in theory, maybe it only contains the keys 'fr' and 'es'. Should still work, it only uses en/nl when present.